src/pages/blog/2024-06-08-announcing-new-graphql-website/index.mdx

```diff
@@ -0,0 +1,123 @@
+
+Later,
+[after 4 years in 2020](https://github.com/graphql/graphql.github.io/pull/896),
+the website was migrated to Gatsby by The Guild members
+[Saihajpreet SINGH](https://x.com/singh_saihaj) and
+[Arda TANRIKULU](https://x.com/ardatanrikulu).
+
+![Screenshot of GraphQL.ORG v1 with Gatsby](./graphql-org-v1.gif)
+
+<p className="text-center">GraphQL.ORG v1 x Gatsby</p>
+
+All of us GraphQL developers spent a lot of time learning this language from the
+"Learn" page, which is shown below, and you definitely remember it!
+
+> Also, we all hated visiting this page at night due to the lack of a night
+> theme. <span className="not-italic">😅</span>
+
+![Screenshot of GraphQL.ORG v1 learn page](./graphql-org-v1-learn-page.png)
+
+<p className="text-center">GraphQL.ORG Learn page x Gatsby</p>
+
+With the new website, we've made a lot of improvements, including a dark theme,
+below how the "Learn" page looks like:
+
+{/* prettier-ignore */}
+<div className="mt-6 dark:even:[&_img]:hidden odd:[&_img]:hidden dark:odd:[&_img]:block">
+  <>![Screenshot of GraphQL.ORG v2 learn page](./graphql-org-v2-learn-page-light.png)</>
+  <>![Screenshot of GraphQL.ORG v2 learn page](./graphql-org-v2-learn-page-dark.png)</>
+  <p className="text-center">GraphQL.ORG Learn page x Nextra</p>
+</div>
+
+## What's New
+
+Here is a recap of the new features and improvements we've made to the website:
+
+### Migration to Nextra 3 & Vercel
+
+We migrated from Gatsby & Netlify to [Nextra](https://nextra.vercel.app) and
+[Next.js](https://nextjs.org). We chose Nextra because it's a great tool for
+building documentation websites, also [The Guild](https://the-guild.dev)
+maintains and improves it continuously. This migration helped us to improve the
+website's performance and bundle size.
+
+This migration was the final part before releasing the upcoming Nextra version.
+You can read my blog post about Nextra 3 in
+[The Guild blog](https://the-guild.dev/blog/nextra-3). Recently, we migrated all
+The Guild documentation websites to Nextra 3, and we can assume that it's stable
+for production.
+
+Also, at the same time while migrating this website, I started to work on the
+future Nextra 4 version with [app router support](https://nextjs.org/docs/app).
+Once the first Nextra 4 alpha is out, this website be updated too!
+
+### New Search Engine
+
+A search was migrated from Algolia to Nextra's built-in search
+[Flexsearch](https://github.com/nextapps-de/flexsearch)
+
+### Light and Dark Themes
+
+For the first time, since the creation of GraphQL.ORG, we have a dark theme! You
+can switch between light and dark themes using the toggle in the top right
+corner.
+
+### Improved Accessibility
+
+Accessibility and contrast mode have been improved to make the website more
+accessible.
+
+### Better Bundle Size and First Load Time
+
+The new website has a better bundle size and first load time compared to the
+previous version.
```

Comment:
If you have any numbers on this (reduced by X% and now loads Y times faster), that would be a great addition. No worries if not!

Reply:
I tried and I don't know how to properly measure this transition between Gatsby and Next.js, so I just completely removed this paragraph